tensorlayer/layers/convolution/expert_conv.py

```diff
@@ -0,0 +1,228 @@
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+
+from tensorlayer.layers.core import Layer
+from tensorlayer.layers.utils import get_collection_trainable
+
+from tensorlayer import tl_logging as logging
+
+from tensorlayer.decorators import deprecated_alias
+
+__all__ = [
+    'Conv1d',
+    'Conv2d',
+]
+
+
+class Conv1d(Layer):
```

Comment:
this is simplified conv, it seems you switched pro APIs and simplified APIs

Reply:
Oh damned ...
I'll rename the files xD